app/helpers/order_helper.rb

```diff
@@ -0,0 +1,13 @@
+module OrderHelper
+  def get_cart
```

Comment:
it conflicts w/ the order_controller "cart" method. it was recursively calling itself so I renamed this to get_cart.